addhost.php

```diff
@@ -9,6 +9,8 @@
  * @copyright  (C) 2006 - 2012 Adam Armstrong
```

Comment:
Imo we should just delete this file if it's features are ported to device:add now

Reply:
I don't think we can... it has been around too long.

at best we can make it s stub that only prints, please use `lnms device:add`

Reply:
We could symlink to artisan, and then in artisan check what script called it, and route to device:add command.

Reply:
The input for addhost.php is garbage... we can't really handle it nicely no matter what we do.  I prefer this change for now, which seeks to have as few side effects as possible, but still enable legacy code removal.  (after it is removed in other places)

We can definitely do more changes later.

Reply:
I meant so it just prints the help of device:add, but nvm its a bit excessive.
I'm all for replacing it with `echo "Please use lnms device:add instead";`, the project doesn't have the manpower to maintain old backward compatibilities